src/Raven.Server/Documents/Indexes/IndexStore.cs

```diff
@@ -418,6 +418,9 @@ private void HandleChangesForStaticIndexes(DatabaseRecord record, long index, Li
                 var name = kvp.Key;
                 var definition = kvp.Value;
 
+                if (definition.Name.StartsWith(Constants.Documents.Indexing.SideBySideIndexNamePrefix, StringComparison.OrdinalIgnoreCase))
```

Comment:
it is unexpected to have `SideBySideIndexNamePrefix` in the definition that comes from the database record.
Why is this happening?

Reply:
we even assert for that in the `ClusterStateMachine`

Reply:
Because in 'DocumentDatabase.Initialize' eventually calls twice to 'IndexStore.HandleChangesForStaticIndexes' that creates SideBySide index in 'HandleStaticIndexChange' call (IndexStore - line 426).

Reply:
we assert for that in the ClusterStateMachine only when you create new database.